clap_builder/src/builder/arg.rs

```diff
@@ -4349,13 +4362,17 @@ impl Arg {
             rendered.push_str(&arg_name);
         }
 
-        let mut extra_values = false;
-        extra_values |= val_names.len() < num_vals.max_values();
-        if self.is_positional() && matches!(*self.get_action(), ArgAction::Append) {
-            extra_values = true;
-        }
-        if extra_values {
-            rendered.push_str("...");
+        if self.terminator.is_none() {
```

Comment:
This deserves a comment that we don't need to display `...` as it was handled along with the value terminator